### PR TITLE
Vue3 mind math

### PR DESCRIPTION
**Pull request by UtkarshSiddhpura**

### CODE REVIEW

File: activities/MindMath.activity/js/activity.js [hunk 1]
```diff
@@ -289,16 +292,6 @@ const app = Vue.createApp({
 
     },
 
-    localized: function() {
-      this.$refs.SugarTutorial.activityLocalized(this.SugarL10n.localize);
-      document.getElementById('next-slot-text').innerHTML = this.SugarL10n.get("NextSlot");
-      document.getElementById('no-timer-button').innerHTML = this.SugarL10n.get("NoTimer");
-      document.getElementById('easy-button').title = this.SugarL10n.get("EasyLevel");
-      document.getElementById('medium-button').title = this.SugarL10n.get("MediumLevel");
```
Comment (UtkarshSiddhpura) [inline activities/MindMath.activity/js/activity.js:295]: We don't to handle palette strings here as it'll be handled in their respective palette.js file
Hence we also don't have to wait for activity initialization(i.e all the palette are on the DOM to be accessed via document.getEle...)  to trigger "localized" event.
This code: https://github.com/llaske/sugarizer/pull/1381/commits/625c05f3a2770cab07d7c98aee77450ca0ae138a#diff-a6d6d8775bb10d9a02bd716602f205816dd159dcd42e01c89ea6a081a367dcdeL21-L33

File: activities/MindMath.activity/js/components/SugarL10n.js [hunk 2]
```diff
@@ -1,88 +1,91 @@
...
-	},
-	watch: {
-		readyToEmit: function (newVal, oldVal) {
-			if (newVal) {
-				this.$emit("localized");
-				this.eventReceived = true;
-			}
-		}
+		};
 	},
```
Comment (UtkarshSiddhpura) [inline activities/MindMath.activity/js/components/SugarL10n.js:18]: This watcher is not required as explained above.

File: activities/MindMath.activity/js/components/SugarL10n.js [hunk 3]
```diff
@@ -1,88 +1,91 @@
...
-			return (this.dictionary != null) && this.activityInitialized;
-		}
-	},
-	watch: {
-		readyToEmit: function (newVal, oldVal) {
-			if (newVal) {
-				this.$emit("localized");
-				this.eventReceived = true;
-			}
-		}
```
Comment (UtkarshSiddhpura) [inline activities/MindMath.activity/js/components/SugarL10n.js:32]: This code is no longer needed as explained above

**Comment by llaske** [comment]
I'm okay with the localization component but there is still lot of issues in the console:

image

File: activities/MindMath.activity/js/activity.js [hunk 4]
```diff
@@ -176,9 +180,9 @@ const app = Vue.createApp({
       if (vm.slots[vm.qNo].length != 0) {
         currentRes = vm.slots[vm.qNo][vm.slots[vm.qNo].length - 1].res;
         if (vm.compulsoryOpsRem.length === 0 && currentRes === vm.questions[vm.qNo].targetNum) {
-          return true;
           //notifying user
           vm.sugarPopup.log("You Got the Target Number")
```
[the remark this answers is not in the document]
Comment (UtkarshSiddhpura) [reply activities/MindMath.activity/js/activity.js:184]: So should I remove the whole popup message? "You Got the ....Number?"

**Review by UtkarshSiddhpura — COMMENTED** [review]
..

**Comment by llaske** [comment]
Merged in https://github.com/llaske/sugarizer/commit/536d13c3b55f33766a90d5e0c08463cba8701ec7

**Pull request closed without merging**